library/src/main/java/com/leinardi/android/speeddial/SpeedDialView.java

```diff
@@ -831,7 +791,7 @@ private void visibilitySetup(boolean visible, boolean animate, boolean reverseAn
 
     private void showWithAnimationFabWithLabelView(FabWithLabelView fabWithLabelView, int delay) {
         ViewCompat.animate(fabWithLabelView).cancel();
-        enlargeAnim(fabWithLabelView.getFab(), delay);
+        UiUtils.enlargeAnim(getContext(), fabWithLabelView.getFab(), delay);
```

Comment:
There is no need to send another instance of the Context, the second parameter, a View, has already a Context. Please get the context from the View and remove the first parameter (see line 175).

- [ ] get the Context from the View